### PR TITLE
[sdks] Create archive targets for pre-building on CI

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

File: scripts/ci/run-jenkins.sh [hunk 1]
```diff
@@ -98,18 +98,14 @@ if [[ ${CI_TAGS} == *'product-sdks-ios'* ]];
...
 		   echo "ENABLE_CXX=1" >> sdks/Make.config
 	   fi
 	   export device_test_suites="Mono.Runtime.Tests System.Core"
 
-	   ${TESTCMD} --label=provision-llvm --timeout=60m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds provision-llvm36-llvm32 provision-llvm-llvm64
-
-	   ${TESTCMD} --label=build-sim-runtimes --timeout=60m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds package-ios-{sim64,sim32,simtv,simwatch}
-	   ${TESTCMD} --label=build-dev-runtimes --timeout=60m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds package-ios-{target64,target32,targettv,targetwatch}
-	   ${TESTCMD} --label=build-cross-compilers --timeout=60m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds package-ios-{cross64,cross32,cross32-64,crosswatch}
+	   ${TESTCMD} --label=archive --timeout=180m --fatal make -j ${CI_CPU_COUNT} -C sdks/builds archive-ios NINJA=
```
Comment (marek-safar) [inline scripts/ci/run-jenkins.sh]: Such timeouts are scary

Comment (lewurm) [reply scripts/ci/run-jenkins.sh]: it's the previous three steps squashed into one step, so it doesn't change anything really.

**Comment by luhenry** [comment]
@monojenkins backport to 2018-08

**Comment by luhenry** [comment]
@monojenkins backport to 2018-10

**Pull request merged**